### ISSUE TITLE
[UI] : Enhance Home page

### ISSUE THREAD

**Issue by anand-harsh** [anand-harsh/Edumi#56]
## Issue
The text and button is having is not in center of page with proper font and also the logo is shifted to right.

## Solution
Do make the text and button to proper size and make them to center with proper alignment and gap

![image](https://github.com/anand-harsh/Edumi/assets/94885893/b8d77720-4465-4948-9dba-c424dd43ce5e)

**Comment by nandita27iitp** [comment]
@anand-harsh please assign this issue to me under JWOC

**Comment by nandita27iitp** [comment]
![image](https://github.com/anand-harsh/Edumi/assets/94813932/a5e72f9e-4f70-4f21-8350-4a395ab930f2)
@anand-harsh pls review it

**Comment by anand-harsh** [comment]
> ![image](https://github.com/anand-harsh/Edumi/assets/94813932/a5e72f9e-4f70-4f21-8350-4a395ab930f2)
> @anand-harsh pls review it

No, the text and logo will be aligned horizontally with proper gap as it was earlier

Just align all component to centre of page
And give proper dont to text and button

Also check for responsive in small devices

**Comment by nandita27iitp** [comment]
![image](https://github.com/anand-harsh/Edumi/assets/94813932/97df4afe-30aa-4340-a2c9-f1bcfc59c28e)
@anand-harsh Pls review

**Comment by anand-harsh** [comment]
> ![image](https://private-user-images.githubusercontent.com/94813932/297112378-97df4afe-30aa-4340-a2c9-f1bcfc59c28e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MDU0MjEzNzgsIm5iZiI6MTcwNTQyMTA3OCwicGF0aCI6Ii85NDgxMzkzMi8yOTcxMTIzNzgtOTdkZjRhZmUtMzBhYS00MzQwLWEyYzktZjFiY2ZjNTljMjhlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAxMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMTE2VDE2MDQzOFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTkwMjUwOThmNjBiNjg4MzdjYzA0OTgxNTE5ZTI0Yzg0MDlkNzA5MTE2MTBiZGMyODUxYjFhMTYzYWZjMzU2YmYmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.bhWT-bd7FEb4GcmNgHS7nlLtN-9n0NtGiiDu1eWSCcM) @anand-harsh Pls review

Yes

**Comment by nandita27iitp** [comment]
Should i PR

**Comment by anand-harsh** [comment]
> Should i PR

yes, have a PR, rest updation can be made there
